hist/hist/src/TH1.cxx

```diff
@@ -3819,8 +3819,8 @@ TFitResultPtr TH1::Fit(const char *fname ,Option_t *option ,Option_t *goption, D
 /// Fit histogram with function f1.
 ///
 /// \param[in] option fit options is given in parameter option.
-///        - "W"  Set all weights to 1 for non empty bins; ignore error bars
```

Comment:
I thought we are explicitly ignoring uncertainties when this option is passed (no normalization of the difference by the uncertainty)? The new doc makes it sound as if the bin uncertainties are actually modified? (Not that the old doc was better, but...)

Reply:
Probably there is no need to mention that the bin uncertainty are set to 1. It is probably sufficient to write something like: "Ignore the bin uncertainties when fitting using the default chi2  method"

Reply:
Same for the other occurrences (TGraph) I suppose?

Reply:
Yes, also there.